les_14/hw_les_14/task_6_hw_test.py

```diff
@@ -0,0 +1,67 @@
+# На семинаре 13 был создан проект по работе с
+# пользователями (имя, id, уровень).
+# Напишите 3-7 тестов pytest для данного проекта.
+# Используйте фикстуры.
+
+import re
+import pytest
+from os import path
+from Project import Project
+from User import User
+import Excep as Ex
+
+@pytest.fixture
+def project():
+    return Project.load_users('users.json')
+
+@pytest.fixture
+def admin(project):
+    project.enter('Sam', 114)
+    return project.admin
+
+def test_enter(project):
+    project.enter('Han', 105)
+    assert project.admin.name == 'Han' and \
+            project.admin.uid == 105
```

Comment:
И это можно использовать тут
```suggestion
def test_enter(project, admin):
    project.enter(admin)
    assert project.admin.name == 'Sam' and \
            project.admin.uid == 144
```
Хотя данные для фикстуры admin могут быть тоже получены из фикстуры